audbackend/core/backend.py

```diff
@@ -489,31 +493,23 @@ def put_archive(
             FileNotFoundError: if ``src_root``,
                 ``tmp_root``,
                 or one or more ``files`` do not exist
-            RuntimeError: if extension of ``dst_path`` is not supported
+            RuntimeError: if ``dst_path`` does not end with
+                ``zip`` or ``tar.gz``
+                or a file in ``files`` is not below ``root``
             ValueError: if ``dst_path`` contains invalid character
                 or does not start with ``'/'``
 
         Examples:
             >>> backend.exists('/a.tar.gz', '1.0.0')
             False
-            >>> backend.put_archive('.', ['src.pth'], '/a.tar.gz', '1.0.0')
+            >>> backend.put_archive('.', '/a.tar.gz', '1.0.0')
             >>> backend.exists('/a.tar.gz', '1.0.0')
             True
 
         """
         dst_path = utils.check_path(dst_path, self.sep)
         src_root = audeer.path(src_root)
 
-        if not os.path.exists(src_root):
-            utils.raise_file_not_found_error(src_root)
-
-        files = audeer.to_list(files)
-
-        for file in files:
-            path = os.path.join(src_root, file)
-            if not os.path.exists(path):
-                utils.raise_file_not_found_error(path)
```

Comment:
I guess you were able to remove this code as those errors are now raised by `audeer.create_archive()`?

Reply:
yes